### DISCUSSION TITLE
French keyboard mapping?

### DISCUSSION THREAD

**Discussion by gregfr** [kovidgoyal/kitty#6713] [General]
Greetings!
I just discovered Kitty :cat: and it looks amazing! However I'm using a French keyboard and some documented shortcuts don't apply ; do you have a setup/cheatsheet for French keyboards?
Thanks in advance
Regards

**Comment by kovidgoyal** [comment]
No, I'm afraid there isnt one, sorry.

**Reply by gregfr** [reply]
Thanks :)
